.github/workflows/test.yml

```diff
@@ -31,6 +31,7 @@ jobs:
         run: |
           excludelist="$(find ./ -type f -name '*.go' | xargs grep -l 'DONTCOVER')"
           excludelist+=" $(find ./ -type f -name '*.pb.go')"
+          excludelist+=" $(find ./ -type f -name '*.pb.gw.go')"
```

Comment:
does not seem to work in the coverage report that was commented in the PR

Reply:
<img width="694" alt="Screenshot 2022-08-24 at 5 59 24 PM" src="https://user-images.githubusercontent.com/10805402/186418602-1d677fa4-f7f1-492c-b453-734f50530c6b.png">

Reply:
resolved